### PR TITLE
auto-set sort direction on sort type change

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Sets ASC/DESC automatically on changing the sort type. 

Do we want this behavior like this by default? Curious for opinions

### CODE REVIEW

**Review by amy-corson-ibigroup — CHANGES_REQUESTED** [review]
On a new search, when I set d/t to "depart" I'm consistently getting this blank dropdown 
image

**Comment by miles-grant-ibigroup** [comment]
Blocked by https://github.com/opentripplanner/otp-react-redux/pull/1316

**Comment by miles-grant-ibigroup** [comment]
All is fixed after latest merge!

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good!

File: lib/components/form/date-time-modal.tsx
Comment (binh-dam-ibigroup) [inline lib/components/form/date-time-modal.tsx]: Maybe rename to `DepartArriveDefaultSortMap` or something like this, to clarify it is a default setting?

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**
